README.md

```diff
@@ -9,3 +9,8 @@
 Wall-E, named after the lovable character [Wall-E](https://en.wikipedia.org/wiki/WALL-E), is the CSSS Discord Bot. This bot is owned by the CSSS and will be maintained by the Bot_manager team.  
 
 What commands do we have? You can find this out by looking at the file located at the following directory [(`commands_to_load/help.json`)](https://github.com/CSSS/wall_e/blob/master/commands_to_load/help.json) or log onto the [discord server](https://discord.gg/9jZ5wa8?fbclid=IwAR3rKhuygMViNlOH0CoLmIloA-X-5RjIelevzIj25uwEHq-RimIb5T1xWYs) and run the command `.help`  
+
+
+### Additional Resources
```

Comment:
this README is not for dev work instructions, dev work should be in the `How to work on the bot` README